### ISSUE TITLE
swe_split_deg throws exception on web

### ISSUE THREAD

**Issue by timashev** [vm75/sweph.dart#9]
When running on web, this code:
```flutter
final long = Sweph.swe_split_deg(
    100,
    SplitDegFlags.SE_SPLIT_DEG_ZODIACAL |
        SplitDegFlags.SE_SPLIT_DEG_ROUND_SEC |
        SplitDegFlags.SE_SPLIT_DEG_KEEP_SIGN |
        SplitDegFlags.SE_SPLIT_DEG_KEEP_DEG);
```
or this:
```
final lat = Sweph.swe_split_deg(
    11.12,
    SplitDegFlags.SE_SPLIT_DEG_ROUND_SEC |
        SplitDegFlags.SE_SPLIT_DEG_KEEP_DEG);
```
always throws:
```
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 288:49      throw_
dart-sdk/lib/_internal/js_dev_runtime/private/native_typed_data.dart 475:5        getInt64]
packages/wasm_ffi/src/ffi/extensions.dart 162:49                                  [as Int64Pointer.[]]
packages/wasm_ffi/src/ffi/extensions.dart 151:24                                  Int64Pointer.get$35value
packages/sweph/sweph.dart 1426:39                                                 <fn>
packages/sweph/src/utils.dart 111:31                                              using
packages/sweph/sweph.dart 1418:12                                                 swe_split_deg
```

Using Flutter 3.10.6 on macOS 13.4.1, Chrome 114.0.5735.198 (arm64)

**Comment by vm75** [comment]
Thanks for reporting this. I will debug and get back.

**Comment by vm75** [comment]
fixed in version 2.10.3+13
